pandasai/llm/google_vertexai.py

```diff
@@ -115,6 +126,8 @@ def _generate_text(self, prompt: str) -> str:
                 top_k=self.top_k,
                 max_output_tokens=self.max_output_tokens,
             )
+        else:
+            raise UnsupportedOpenAIModelError("Unsupported model")
```

Comment:
Shouldn't we generalize this exception? Or just throwing a `ValueError` would suffice, I suppose.

Reply:
Yep, correct! Done @mspronesti